source/walkthroughs/deploy/deploy_app/_preamble.md.erb

```diff
@@ -118,11 +118,11 @@ substep = 0
     In order to provide you appropriate instructions, please choose your Meteor version: <%= render_partial("/shared/meteor_picker.html") %>
   </p>
   <p>
-    Inside your application's code directory, on your local computer, use the <code class="meteor old">meteor bundle</code><code class="meteor new">meteor build</code> command to create a Meteor package tarball.
+    Inside your application's code directory, on your local computer, use the <code class="meteor v13">meteor bundle</code><code class="meteor v14 v16">meteor build</code> command to create a Meteor package tarball.
   </p>
 
-  <pre class="highlight meteor old"><span class="prompt">$ </span>meteor bundle package.tar.gz</pre>
-  <pre class="highlight meteor new"><span class="prompt">$ </span>meteor build --server-only ../new_package && mv ../new_package/*.tar.gz package.tar.gz</pre>
+  <pre class="highlight meteor v13"><span class="prompt">$ </span>meteor bundle package.tar.gz</pre>
+  <pre class="highlight meteor v14 v16"><span class="prompt">$ </span>meteor build --server-only ../new_package && mv ../new_package/*.tar.gz package.tar.gz</pre>
```

Comment:
Hmm, well, nice suggestions, makes sense to me, although I can make some arguments (_to at least make this suggestion an additional PR to track separately_):

* The current [initial step to select infrastructure](https://www.phusionpassenger.com/library/walkthroughs/deploy/meteor/) already deals only with linux deployments and there's a [small note for 32-bit deployments](https://www.phusionpassenger.com/library/walkthroughs/deploy/meteor/ownserver/nginx/oss/install_language_runtime.html) 
* Non-linux-64bit targets might be discussed as an option on some other page perhaps because
  * There's no windows deployment option with passenger
  * Meteor's linux 32bit support (and interest) is arguably fading
  * I don't think osx is popular for production deployments
* Perhaps `--architecture=os.linux.x86_64` might be added here by default, assuming that it would be the most common case
* Updating the current markup partial inclusion and accompanied jquery magic to also incorporate this option would become unwieldy and I'd say we should find some other way to convey this information/choice

Reply:
Agree on separate PR and difficulty with jQuery magic, mb @OnixGH can point to the better solution?